SwrveSDK/SwrveStorageManager.brs

```diff
@@ -138,15 +97,16 @@ function SwrveIsResourceFileValid() as Boolean
 	if persistentSignature = SwrveMd5(persistentResource)
 		return true
 	else
-		SWLog("Resource file has been compromised. Reloading.")
+		SWLogWarn("Resource file has been compromised. Reloading.")
 		return false
 	end if
 end function
 
 function SwrveClearWholePersistence()
-	ro = CreateObject("roRegistry")
-    for each section in ro.GetSectionList()
-        ro.delete(section)
-    end for
-    ro.flush()
-end function
\ No newline at end of file
+	' TODO: Update to only clear Swrve sections
```

Comment:
Would like to maybe chat about this as it seems concerning that this API would also result in the deletion of channel registry values. This likely should be scoped to only clearing Swrve data.

Reply:
Yeah, not for public use. Testing function, used by unit tests in private repo. We'll remove or alter.